### ISSUE TITLE
Target directory sometimes missing from mount entry

### ISSUE THREAD

**Issue by talyz** [mpartel/bindfs#94]
Most of the time, `mount` entries for `bindfs` contain the target directory:
```
/persistent/home/talyz/.local/share/direnv on /home/talyz/.local/share/direnv type fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=100,default_permissions)
```
but sometimes, they don't:
```
bindfs on /home/talyz/VirtualBox VMs type fuse.bindfs (rw,nosuid,nodev,relatime,user_id=1000,group_id=100,default_permissions)
```
We're using `bindfs` in the [nix-community/impermanence](https://github.com/nix-community/impermanence) project to bind mount state from persistent storage into the user's home directory, which is otherwise ephemeral. We need the current target directory in order to know whether it's up-to-date when switching from an old to a new configuration.

**Comment by mpartel** [comment]
bindfs doesn't show the source dir there if it contains whitespace or a comma. I don't remember (and failed to document) whether/why the whitespace check was strictly necessary, but at least a [comma messed up FUSE's option parser](https://github.com/mpartel/bindfs/issues/47).

I released 1.14.9 where you can pass `-o fsname=$X` to show `$X` in the mount entry. An `$X` containing whitespace *seems* to work.

Please feel free to reopen if this doesn't solve your problem.

**Comment by talyz** [comment]
This is perfect! Thank you so much for the quick fix!
